task/isto-canary-release/0.1/istio-canary-release.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: istio-canary-release
```

Comment:
Can you please change the directory name to `istio-canary-release` to match the resource name